invoke.yml.example

```diff
@@ -2,14 +2,13 @@
 nautobot:
   project_name: "nautobot"
   local: False
-  python_ver: "3.10"
-  compose_dir: "/full/path/to/nautobot/development"
+  python_ver: "3.12"
+  # compose_dir: "/full/path/to/nautobot/development"
   cache_test_fixtures: False
   compose_files:
     - "docker-compose.yml"
     - "docker-compose.postgres.yml"
     - "docker-compose.dev.yml"
-    - "docker-compose.override.yml"
   docker_image_names_main:
```

Comment:
Can remove the docker_image_names_main and docker_image_names_develop keys too.

Really *all* it should do by default is define the `compose_files`, everything else is optional and can be left as a comment.

Reply:
I moved everything to a comment.